core/scripts/cre/environment/environment/setup.go

```diff
@@ -277,24 +312,28 @@ func RunSetup(ctx context.Context, config SetupConfig) error {
 	}
 	logger.Info().Msg("✓ AWS CLI is installed")
 
-	ghCli, ghCliErr := checkGHCli(ctx)
+	ghCli, ghCliErr := checkGHCli(ctx, noPrompt)
 	if ghCliErr != nil {
 		return errors.Wrap(ghCliErr, "failed to ensure GitHub CLI")
 	}
 
-	jdLocalImage, jdErr := JDImageConfig.Ensure(ctx, dockerClient)
+	jdLocalImage, jdErr := JDImageConfig.Ensure(ctx, dockerClient, noPrompt, purge)
 	if jdErr != nil {
 		return errors.Wrap(jdErr, "failed to ensure Job Distributor image")
 	}
-	chipLocalImage, chipErr := ChipImageConfig.Ensure(ctx, dockerClient)
+	chipLocalImage, chipErr := ChipImageConfig.Ensure(ctx, dockerClient, noPrompt, purge)
 	if chipErr != nil {
 		return errors.Wrap(chipErr, "failed to ensure Atlas Chip Ingress image")
 	}
 
-	creCLI, creCliErr := checkCRECLI(ctx)
+	creCLI, creCliErr := checkCRECLI(ctx, noPrompt, purge)
```

Comment:
I think we can at least comment it out if not remove, since with that `workflow deploy` changes we don't need that CLI any more. Also, when we have fetched configured for local file, we cannot use Gist anyway (which makes me think that maybe that `URL = "file:///home/chainlink/workflows"` setting should accept an array, so that we put both local file there and Gist or S3?)